src/main/resources/io/jenkins/plugins/orka/OrkaAgent/configure-entries.jelly

```diff
@@ -80,6 +80,18 @@
     <f:entry title="${%Name Prefix}" field="namePrefix" help="${descriptor.getHelpFile('namePrefix')}">
       <f:textbox/>
     </f:entry>
+
+    <f:entry title="Port Mapping" field="portMapping" description="Please enter the 'from' and 'to' ports for port mapping. Only ports between 1024 and 65535 are allowed.">
```

Comment:
The technical terms for these ports are `host` and `guest`. I suggest we stick to them in the user visible sections

Reply:
In the cli, the terms are NODE_PORT and VM_PORT. Shouldn't we be consistent with what the user sees?

```
-p, --ports strings             (Optional) Forwards traffic from the specified 'NODE_PORT' to the specified 'VM_PORT'.
                                  Provide the port mapping in the following format: '<NODE_PORT>:<VM_PORT>'.You can specify multiple ports by separating them with commas.
                                  IMPORTANT: Mapping the VM and node ports does not automatically open a port on your VM. You need to handle the incoming traffic on the VM separately.
```

Reply:
That's a good call. Let's use node and vm port